.ci/setup_env_centos.sh

```diff
@@ -18,48 +18,43 @@ else
   centos_version=$(grep VERSION_ID /usr/lib/os-release | cut -d '"' -f2)
 fi
 
+[ "$centos_version" == 8 ] || die "This script is for CentOS 8 only"
+
 # Send error when a package is not available in the repositories
 echo "skip_missing_names_on_install=0" | sudo tee -a /etc/yum.conf
 
 # Check EPEL repository is enabled on CentOS
-if [ -z "$(yum repolist | grep 'Extra Packages')" ]; then
-	echo >&2 "ERROR: EPEL repository is not enabled on CentOS."
+if [ -z "$(dnf repolist | grep 'Extra Packages')" ]; then
+	info "EPEL repository is not enabled on CentOS. Going to enable it."
 	# Enable EPEL repository on CentOS
-	sudo -E yum install -y wget rpm
-	wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-${centos_version}.noarch.rpm
-	sudo -E rpm -ivh epel-release-latest-${centos_version}.noarch.rpm
+	sudo -E dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
I'd simply drop the if statement and do a `sudo -E dnf -y install epel-release`.  If it's already installed, it won't fail.  And as we have the package as part of CentOS, I see no good reason to downloading it from somewhere else.

Reply:
Done.